index.html

```diff
@@ -396,21 +396,36 @@
           Install prompts
         </h2>
         <p>
-          An end-user can <dfn data-lt="manual installation">manually</dfn>
+          There are three ways that the installation process can be triggered:
+        </p>
+        <ul>
+          <li>An end-user can <dfn data-lt="manual installation">manually</dfn>
           trigger the <a>installation process</a> through the user agent's
           <abbr>UI</abbr>.
-        </p>
-        <p>
-          Alternatively, the <a>installation process</a> can occur through an
+          </li>
+          <li>The <a>installation process</a> can occur through an
           <dfn>automated install prompt</dfn>: that is, a <abbr>UI</abbr> that
           the user agent presents to the user when, for instance, there are
           sufficient <a>installability signals</a> to warrant
           <a>installation</a> of the web application.
-        </p>
+          </li>
+          <li>The <a>installation process</a> can occur through a
+          <dfn>site-triggered install prompt</dfn>: the site can
+          programmatically request that the user agent present an install
+          prompt to the user. The availability of this feature may be
+          restricted to when, for instance, there are sufficient
+          <a>installability signals</a> to warrant <a>installation</a> of the
+          web application.
+          </li>
+        </ul>
         <p>
           Prior to presenting an <a>automated install prompt</a>, a user agent
-          MUST run the <a>steps to notify before an automated install
-          prompt</a>.
+          MUST run the <a>steps to notify that an install prompt is
+          available</a>, to give the site the opportunity to suppress the
```

Comment:
I wonder if we should say, instead of "to give the site the opportunity to suppress the install prompt" something like "prevent the default action (which is to install the application)". That would maybe align better with `. preventDefault()`